lib/buildInEnums.js

```diff
@@ -0,0 +1,167 @@
+export const predefinedActions = {
+  androidCopyImages: 'android/copyImages',
+  copyAssets: 'copy_assets',
+};
+
+export const logWarningLevels = {
+  warn: 'warn',
+  error: 'error',
+  disabled: 'disabled',
+};
+
+export const logVerbosityLevels = {
+  default: 'default',
+  silent: 'silent',
+  verbose: 'verbose',
+};
+
+export const logBrokenReferenceLevels = {
+  throw: 'throw',
+  console: 'console',
+};
+
+export const commentStyles = {
+  short: 'short',
+  long: 'long',
+  none: 'none',
+};
+
+export const commentPositions = {
+  above: 'above',
+  inline: 'inline',
+};
+
+export const commentStylesFileHeader = {
+  short: 'short',
+  long: 'long',
+  xml: 'xml',
+};
+
+export const fileFormatNames = {
```

Comment:
```suggestion
export const formats = {
```

Reply:
My thoughts on the naming have been, that it should not get confusing, if you ever have an array of format objects somewhere in the code? In order to actually refelect, what to expect behind this variable name, I came up with `fileFormatNames`. Downside, it's admittedly longer.

Reply:
see https://github.com/amzn/style-dictionary/pull/1375#discussion_r1819115164

Reply:
I have renamed that enum not to `formats` but to `fileFormats` now, because there already have been multiple name conflicts with the important `formats` variable defined in `lib/common/formats.js`.